### PR TITLE
Draft decision process

### PR DESCRIPTION
**Pull request by DavidFatDavidF**
Strawman of the decision making process to be discussed in the wg monthly meeting on 20th April 2020

### CODE REVIEW

File: guidelines/decision-process.md [hunk 1]
```diff
@@ -1 +1,23 @@
+# Decision making process
 
+## Preamble
+The Chair Group discussion led to a consensus to start with a light weight consensus-based process that would eventually use voting as a last resort method in case of impasse.. The group could eventually be in need to defend its decision making process against trolls, yet at the moment all group members are considered in good faith, so the working group reserves the option to specify disciplinary actions against trolls as part of its guidelines at a later stage..
```
Comment (nbouvrette) [inline guidelines/decision-process.md:4]: Why the double dots?

> case of impasse..
> later stage..

Comment (DavidFatDavidF) [reply guidelines/decision-process.md:4]: First, not married to the double dots.. feel free to convert them to full stops.
Second, it's in the informal preamble/background piece indicating a topic that can have development, more could be said if necessary..

File: guidelines/decision-process.md [hunk 2]
```diff
@@ -1 +1,23 @@
...
+
+## Definitions
+1. *Consensus* is defined as lack of sustained opposition.
+2. *Good standing* is a characteristic of group members who fullfill their implicit and explicit obligations and hence are allowed to excercise all of their rights within the group without restriction.
+3. *Blacklist* (*blacklisted*, *blacklisting*) is a taxative enumeration of group mebers who are temporarily excluded from excercising their rights within the group.
```
Comment (romulocintra) [inline guidelines/decision-process.md:9]:
```suggestion
3. *Blacklist* (*blacklisted*, *blacklisting*) is a taxative enumeration of group members who are temporarily excluded from exercising their rights within the group.
```

File: guidelines/decision-process.md (hunk 1 above)
Comment (romulocintra) [inline guidelines/decision-process.md:4]: > so the working group reserves the option to specify disciplinary actions against trolls as part of its guidelines at a later stage..

 Not sure about the phrase(if it should belong to another place)

File: guidelines/decision-process.md [hunk 3]
```diff
@@ -1 +1,23 @@
...
+4. *Normative keywords* as defined in [BCP 14](https://tools.ietf.org/html/bcp14)
+
+## Rules
+1. Any current group member is deemed to be in good standing unless blacklisted.
```
Comment (romulocintra) [inline guidelines/decision-process.md:13]: Instead of Rules probably i would call it "Process" what do you think ?

Comment (DavidFatDavidF) [reply guidelines/decision-process.md:13]: I think that it's indeed rules rather than process steps..

File: guidelines/decision-process.md (hunk 1 above)
Comment (DavidFatDavidF) [reply guidelines/decision-process.md:4]: This is an informal preamble explanation of why the blacklisting process is TBD atm

The whole preamble can be deleted, it doesn't add anything normative, it's just to explain why the rules are currently lightweight and partially underspecified.. 

It is intended to help the reader of the normative part to understand why things are the way they are ;-)

Comment (aphillips) [reply guidelines/decision-process.md:4]: I think it is foolish to leave this undefined--having experienced some world-class trolls and/or ill-behaved persons (you haven't lived until you have received death threats online??). We'd be better off to handle this [the way CLDR and ICU do](http://cldr.unicode.org/index/process): with a reference to the formal rules, located [here](http://www.unicode.org/consortium/tc-procedures.html). In practice, voting and such are almost never used in favor of consensus.

Comment (romulocintra) [reply guidelines/decision-process.md:4]: > (you haven't lived until you have received death threats online??). We'd be better off t
😂😂😂😂😂😂😂😂😂


> We'd be better off to handle this [the way CLDR and ICU do](http://cldr.unicode.org/index/process): with a reference to the formal rules, located [here](http://www.unicode.org/consortium/tc-procedures.html). 

Agree!!

**Comment by DavidFatDavidF** [comment]
I removed Preamble.. This was agreed to be our decision making process in the monthly meeting on 20th April 2020.

**Pull request merged**